src/ContractInteract/validation.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Validates that the provided config object exitsts.
+ *
+ * @param {Object} config The configuration object.
+ *
+ * @throws Will throw an error if configuration is missing.
+ */
+const validateConfigExists = (config) => {
```

Comment:
Is this really required when you always also need to check that the keys exist?

Reply:
We could also move that into the key check, yes.